python/samples/concepts/agents/step4_openai_assistant_agent_code_interpreter.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft. All rights reserved.
+import asyncio
+import os
+
+from semantic_kernel.agents.open_ai.azure_open_ai_assistant_agent import AzureOpenAIAssistantAgent
+from semantic_kernel.agents.open_ai.open_ai_assistant_agent import OpenAIAssistantAgent
+from semantic_kernel.agents.open_ai.open_ai_assistant_definition import OpenAIAssistantDefinition
+from semantic_kernel.agents.open_ai.open_ai_service_configuration import (
+    AzureOpenAIServiceConfiguration,
+    OpenAIServiceConfiguration,
+)
+from semantic_kernel.agents.open_ai.open_ai_thread_creation_options import OpenAIThreadCreationOptions
+from semantic_kernel.contents.chat_message_content import ChatMessageContent
+from semantic_kernel.contents.utils.author_role import AuthorRole
+from semantic_kernel.kernel import Kernel
+
+AGENT_NAME = "CodeRunner"
```

Comment:
In .NET, there are two areas for samples:

1. **GettingStartedWithAgents**: A progression of basic agent concepts
2. **Concepts/Agents**: A variety of samples that might be more complex

I see that Python has _GettingStarted_, but perhaps should also introduce _GettingStartedWithAgents_?  This would contain the Step 1, Step 2, etc... samples that should be aligned with .NET.

What you've introduced as Step4 & Step5 are samples from concepts.  Step3 is really Step8

Reply:
We don't have AgentChat behavior yet, so I am adding samples and their respective steps in chronological order as they're produced. I think it would be more confusing to see "step1," "step2," "step8" at this point in time. We can re-order samples once we have parity.

Reply:
We can update these in a next iteration, as we make more functionality available.

Reply:
I've added a `getting_started_with_agents` folder to better track sample parity with dotnet. I've left some of the samples in the samples/concepts/agents folder.